### DISCUSSION TITLE
v.1.0.1

### DISCUSSION THREAD

**Discussion by Lennolium** [Lennolium/PrivacyShield#6] [Announcements]
- __Optimized:__ Applied PrivacyGate Whitelist to the Blocklist, so PrivacyGate is smaller now.
- __New Source Lists:__ Added many more Source Lists to White- & Blocklist. Check IncludedLists.md for details.
- __Adressed Issues:__ Fixed all false positives which you guys reported in the issues tab. Thanks for your support! <3

<hr /><em>This discussion was created from the release <a href='https://github.com/Lennolium/PrivacyShield/releases/tag/v.1.0.1'>v.1.0.1</a>.</em>

**Comment by malikshi** [comment]
Hi sir, thanks you for great works.
how frequent is the list updated?
weekly or monthly or even daily, since i don't see any code(GA/CI) related how the list updated.

**Reply by malikshi** [reply]
@Lennolium can you answer?

**Reply by Lennolium** [reply]
Sorry for my late answer, I was on vacation.
Right now I'm still at optimizing and finalizing my setup/workflow, so that's why the updates are not so frequently.
After that, updates will be pushed min. weekly.
